troubleshooting-1.Rmd

```diff
@@ -24,14 +26,18 @@ Now, let's load the `tidyverse` (meta-) package:
 
 ```{r, error = TRUE}
 ## ERROR 2 ##
-libraries(tidyverse)
+# Line 30 Max: Wrong function for accessing the package, should be library()
+# libraries(tidyverse)
+library(tidyverse)
 ```
 
 By loading the tidyverse, a function called `glimpse` has been made available. This function is useful for viewing a data set. Let's take a look at the `mtcars` dataset by applying the `glimpse` function to `mtcars`!
 
 ```{r, error = TRUE}
 ## ERROR 3 ##
-glimpse mtcars
+# Line 39 Max: Wrong way to apply functions to dtl, should be glimpse()
```

Comment:
What is 'dtl'? (psych major here). We could also add a comment: we changed "glimpse mtcars" to "glimpse(mtcars)" (The object to which the function is being applied (here: mtcars) should be bracketed beside the function. Thank you Max!

Reply:
changed, have a look.